Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -1847,63 +1847,7 @@ public void DefaultRepeatCountIsThree()
             Assert.Equal<int>(expectedRepeatCount, result);
             // Teardown
         }
-

```

Comment:
Wouldn't it be possible to avoid changing/moving these tests?

AFAICT, this change is a pure refactoring, it shouldn't be necessary to modify existing tests. This also makes it possible to verify that no breaking changes are introduced.

Since `Repeat` is extracted into an `IFixture` extension method, `sut` should still be able to access it, so these tests should still compile and pass.

What do I probably missed? :flushed: